src/d2r-map.ahk

```diff
@@ -406,7 +406,6 @@ MapSizeDecrease:
     return
 }
     
-#IfWinActive, ahk_exe D2R.exe
```

Comment:
Why is this removed? Don't we only want these functions to execute when a D2R.exe window is active?

Reply:
They're inherently active through the hotkey. there are some hot cubes that are defined under that area, such as tab and space. The problem is is that the #if is not dynamic and is executed before anything else is executed in the script, So even though they're only active when that window is open that code cannot be dynamic. So by defining the hotkeys with the ID,  They will only be active on the client they are assigned to, and not just any client that is active.